chart/README.md

```diff
@@ -0,0 +1,134 @@
+# Gitpod Self-Hosted
+
+[gitpod.io](https://gitpod.io) is a service that provides developers with ready-to-code dev environments in the cloud.
+
+This [Helm](https://helm.sh) chart allows you to deploy and operate an instance of Gitpod on your own infrastructure.
+
+
+## Prerequisites
+
+- Kubernetes 1.13+
+- Helm 3+
+
+
+## Get Repo Info
+
+```console
+helm repo add charts.gitpod.io https://charts.gitpod.io
+helm repo add stable https://charts.helm.sh/stable
+helm repo add stable https://helm.min.io/
+helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+
+## Install Chart
+
+```console
+# Helm
+$ helm install $(for i in $(cat configuration.txt); do echo -e "-f $i"; done) gitpod .
```

Comment:
```suggestion
$ helm install gitpod gitpod
```

we don't have the `configuration.txt` anymore, do we?

Reply:
See description above:
> the configuration.txt mechanism has been kept for better modularity

I've not too strong of an opinion on that, though.

Reply:
removed